include/NovelRT/Maths/GeoVector2F.h

```diff
@@ -21,157 +24,789 @@ namespace NovelRT::Maths
+         * @returns true if any of the components do not match, otherwise false.
+         */
         inline bool operator!=(GeoVector2F other) const noexcept
         {
             return *reinterpret_cast<const glm::vec2*>(this) != *reinterpret_cast<const glm::vec2*>(&other);
         }
 
+        /**
+         * @brief Evaluates this GeoMatrix4x4F with another GeoMatrix4x4F to determine if any of it's components is less
+         * than the other's components.
+         *
+         * @param other The GeoMatrix4x4F who's components are evaluated against this GeoMatrix4x4F's components.
+         * @return true if any of the components is less than the other GeoMatrix4x4F's components, otherwise false.
+         */
         inline bool operator<(GeoVector2F other) const noexcept
         {
             return glm::any(
                 glm::lessThan(*reinterpret_cast<const glm::vec2*>(this), *reinterpret_cast<const glm::vec2*>(&other)));
         }
 
+        /**
+         * @brief Evaluates this GeoMatrix4x4F with another GeoMatrix4x4F to determine if any of it's components is less
+         * than or equal to the other's components.
+         *
+         * @param other The GeoMatrix4x4F who's components are evaluated against this GeoMatrix4x4F's components.
+         * @return true if any of the components is less than or equal to the other GeoMatrix4x4F's components,
+         * otherwise false.
+         */
         inline bool operator<=(GeoVector2F other) const noexcept
         {
             return glm::any(glm::lessThanEqual(*reinterpret_cast<const glm::vec2*>(this),
                                                *reinterpret_cast<const glm::vec2*>(&other)));
         }
 
+        /**
+         * @brief Evaluates this GeoMatrix4x4F with another GeoMatrix4x4F to determine if any of it's components is
+         * greater than the other's components.
+         *
+         * @param other The GeoMatrix4x4F who's components are evaluated against this GeoMatrix4x4F's components.
+         * @return true if any of the components is greater than the other GeoMatrix4x4F's components, otherwise false.
+         */
         inline bool operator>(GeoVector2F other) const noexcept
         {
             return glm::any(glm::greaterThan(*reinterpret_cast<const glm::vec2*>(this),
                                              *reinterpret_cast<const glm::vec2*>(&other)));
         }
 
+        /**
+         * @brief Evaluates this GeoMatrix4x4F with another GeoMatrix4x4F to determine if any of it's components is
+         * greater than or equal to the other's components.
+         *
+         * @param other The GeoMatrix4x4F who's components are evaluated against this GeoMatrix4x4F's components.
+         * @return true if any of the components is greater than or equal to the other GeoMatrix4x4F's components,
+         * otherwise false.
+         */
```

Comment:
Ah ok looks like this all suffered the same fate as my first matrix documentation comment above - please fix this.